src/main/kotlin/edu/cornell/cs/apl/viaduct/util/SetComparator.kt

```diff
@@ -0,0 +1,30 @@
+package edu.cornell.cs.apl.viaduct.util
```

Comment:
Comparing sets already does this.

Reply:
there's no `compareTo` method even for `SortedSet`s